src/TextField/TextField.js

```diff
@@ -2,9 +2,19 @@
 
 import React, { Component, PropTypes } from 'react';
 import shallowEqual from 'recompose/shallowEqual';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
 import { Input, InputLabel } from '../Input';
 import FormControl from '../Form/FormControl';
 
+export const styleSheet = createStyleSheet('TextField', () => {
+  return {
+    root: {
+      marginTop: 16,
```

Comment:
`TextField` is higher order than the simple inputs. It's intent as I understand it is to orchestrate the spec functionality of the text field.  While it would be a zero impact for me to remove it, it may be nice to have for others that want all-in-one.  

I use the subcomponents in a similar but customized fashion (adding helper texts) so I don't use `TextField` directly.  Are you sure you still want it removed?